src/raw.rs

```diff
@@ -56,6 +56,24 @@ impl<T> Vec<T> {
         }
     }
 
+    pub fn clear(&mut self) {
+        self.count.store(0, Ordering::Relaxed);
+        self.inflight.store(0, Ordering::Relaxed);
+
+        for (i, b) in self.buckets.iter_mut().enumerate() {
+            let e = b.entries.load(Ordering::Relaxed);
+            if e.is_null() {
+                continue;
+            }
+
+            let len = Location::bucket_capacity(i);
+            for i in 0..len {
+                let e = unsafe { &mut *e.add(i) };
+                e.active.store(false, Ordering::Relaxed);
```

Comment:
This will not drop the values. You can use the raw iterator and call [`next_owned`](https://github.com/ibraheemdev/boxcar/blob/0963e7897bcac37cdd396f290bd0ada9a697f460/src/raw.rs#L512) instead.

Reply:
Not sure how to utilize iterator here, so just dropped the entry similarly to `impl Drop for Entry`.

Also covered that in tests, thanks for noticing.